### PR TITLE
Ui makeover

### PR DESCRIPTION
**Pull request by gvonkoss**
WIP!

### CODE REVIEW

File: views/partials/home.handlebars [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<button class="o-buttons" onclick="location.href='./delete'">Delete Content</button>
+<button class="o-buttons" onclick="location.href='./reingest'">Reingest Content</button>
+<button class="o-buttons" onclick="location.href='./tme'">Reingest Tag</button>
+<button class="o-buttons" onclick="location.href='./date'">Date</button>
```
Comment (i-like-robots) [inline views/partials/home.handlebars]: Make these links?

File: views/partials/delete.handlebars [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<h2>Delete Content</h2>
+<label for="o-forms-standard" class="o-forms__additional-info">Remember to delete content from <span>both</span> clusters!</label>
```
Comment (i-like-robots) [inline views/partials/delete.handlebars]: This should just be a paragraph

File: views/partials/delete.handlebars [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+<input type="text" id="uuid" placeholder="The uuid thing to delete" class="o-forms__text"/>
```
Comment (i-like-robots) [inline views/partials/delete.handlebars]: and the contents of the placeholder should be the label =]

File: views/partials/delete.handlebars [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+<br></br>
+
+<button class="o-buttons deleteItem">Delete</button>
+<button class="o-buttons" onclick="location.href='./home'">Back</button>
```
Comment (i-like-robots) [inline views/partials/delete.handlebars]: linkedy link

File: views/partials/delete.handlebars (hunk 3 above)
Comment (gvonkoss) [inline views/partials/delete.handlebars]: But the place holder is uuid, isn't it?

Comment (i-like-robots) [inline views/partials/delete.handlebars]: Labels describe the input, a placeholder is an example, E.G.:

```html
<label for="email-field">Email address</label>
<input id="email-field" placeholder="you@example.com">
```

So I think should be:

```html
<p>Remember to delete content from <span>both</span> clusters!</p>
<label for="uuid">UUID of content to delete</label>
<input placeholder="00000000-0000-0000-000000000000">
```

May also be worth validating the format (`pattern="[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}"`)

File: views/index.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>{{title}}</title>
+		<script src="//cdn.polyfill.io/v2/polyfill.min.js?features=fetch"></script>
```
Comment (i-like-robots) [inline views/index.html]: I wonder if we should cut-the-mustard and display an error if a browser is unsupported, what do you think?

**Comment by i-like-robots** [comment]
Thanks for your hard work on this! Looks really good, just a couple of small comments

File: views/index.html (hunk 5 above)
Comment (gvonkoss) [inline views/index.html]: I put in the ctm, just not 100% sure it is right, or how to test it locally. Thoughts?

**Comment by i-like-robots** [comment]
👍

**Pull request merged**